.gitignore

```
@@ -8,3 +8,4 @@ target
 .settings
 bin
 .DS_Store
+.vscode/*
```

NIT, easier to block the whole folder instead of files inside 👍 
```suggestion
.vscode/
```